docs/src/cli-reference/TensileCreateLibrary.rst

```diff
@@ -84,6 +86,20 @@ Options
     Output master solution index in csv format including number 
     of kernels per architecture post build in csv format.
 
+Arguments
+---------
+
+When invoking *TensileCreateLibrary*, the following arguments are required.
+
+\<LOGIC DIRECTORY\>
+    Absolute path for logic files. These files are generally found in one of two ways: (i) they are
+    generated via the `Tensile` program and placed in the build directory under *3_LibraryLogic* (see :ref:`quick-start-example`).
+    (ii) they are found within a project that hosts pre-generated logic files, e.g., `rocBLAS <https://github.com/ROCm/rocBLAS/tree/develop/library/src/blas3/Tensile/Logic>`_.
+\<OUTPUT DIRECTORY\>
+    Absolute or relative path to the output directory where build artifacts are placed.
+\<RUNTIME LANGUAGE\>
+    One of: {OCL, HIP, HSA}
+
```

Comment:
I don't know if these qualify as options. We should maybe have a separate section for these.